request.el

```diff
@@ -948,7 +910,20 @@ as the `--user' option."
    (list request-curl
          "--silent" "--location"
          "--cookie" cookie-jar "--cookie-jar" cookie-jar)
-   (request--auth auth url)
+   (when auth
+     (let* ((host (url-host (url-generic-parse-url url)))
+            (auth-source-do-cache nil)
+            (auth-source-creation-prompts '((user . (format "%s user: " host))
+                                            (secret . "Password for %u: ")))
```

Comment:
This doesn't seem to do the right thing. I get the impressionthat format isn't being interpolated correctly. This works:
```
            (auth-source-creation-prompts '((user . "User: ")
                                            (secret . "Password: ")))
```

Reply:
Correct.  Missing backquote-comma in the interpolated list.